### ISSUE TITLE
Missing/Outdated compilation steps

### ISSUE THREAD

**Issue by petitlapin** [Jmgr/actiona#168]
Hi,

the documentation in https://wiki.actiona.tools/doku.php?id=en:compilation and https://wiki.actiona.tools/doku.php?id=en:devel refers to qmake, not cmake.

I'm on Archlinux, to compile, I had to:
* git clone https://github.com/Jmgr/actiona.git
* cd actiona
* git submodule init && git submodule update
* mkdir build && cd build
* cmake .. && make -j 4

I also had to install vtk (`/usr/bin/ld: /usr/lib/libopencv_viz.so.4.5.5: undefined reference to `vtkDataObject::DATA_OBJECT()'`) and pugixml (`/usr/bin/ld: /usr/lib/libvtkCommonDataModel.so.1: undefined reference to `pugi::xml_attribute::set_value(unsigned int)'`) but they are not listed as dependency anywhere?

Cheers

**Comment by Jmgr** [comment]
Yes; cmake has been a recent addition and the documentation has not been updated in a while. Thanks for the build instructions on Archlinux.

> I also had to install vtk (`/usr/bin/ld: /usr/lib/libopencv_viz.so.4.5.5: undefined reference to `vtkDataObject::DATA_OBJECT()'`) and pugixml (`/usr/bin/ld: /usr/lib/libvtkCommonDataModel.so.1: undefined reference to `pugi::xml_attribute::set_value(unsigned int)'`) but they are not listed as dependency anywhere?

Those must be indirect dependencies from OpenCV or Qt, as I don't recall using them directly.

**Comment by petitlapin** [comment]
yes, there are indirect dependencies from opencv (/usr/lib/libopencv_viz.so and pugixml is a dependency of vtk).

I managed to not need them by changing [${OpenCV_LIBS}](https://github.com/Jmgr/actiona/blob/e0c9ef1380e198f8984b807244db8afd3454fb3d/actiontools/CMakeLists.txt#L286) with:
```
opencv_core
opencv_shape
```
I could also remove the `${OpenCV_INCLUDE_DIRS}` above as the targets implicitly add it.

I'm using opencv4, and there are existing targets for cmake. I'm not sure if there is a minimal expected version of opencv and if on the previous versions, their cmake also export these targets?

If that's the case, it should be better as it won't pull all opencv librairies but only the needed ones.

**Comment by Jmgr** [comment]
The documentation has been updated: https://wiki.actiona.tools/doku.php?id=en:compilation
